corehq/apps/app_manager/templates/app_manager/partials/nav_menu_media_single_type.html

```diff
@@ -83,14 +85,20 @@
         <div class="col-sm-4">
             <input type="text" class="form-control"
                    data-bind="value: customPath,
-                              enable: enabled,
-                              valueUpdate: 'textchange'" />
+                         enable: enabled,
+                         valueUpdate: 'textchange'" />
             <input type="hidden" class="jr-resource-field"
                    name="{{ qualifier|default_if_none:"" }}media_{{ type }}"
                    data-bind="value: savedPath" />
-            <input type="hidden"
-                   name="use_default_{{type}}_for_all"
-                   data-bind="value: languagesLinked" />
+            {% if request|toggle_enabled:'LANGUAGE_LINKED_MULTIMEDIA' %}
```

Comment:
Nitpick, but I wouldn't check for the flag here, seems like it could cause unexpected behavior if you're using the flag, then turn it off, then later back on, and find some of your old checkboxes have changed value.

Reply:
you are right, this is a Very Bad way of doing this.